site/docs/v1/tech/apis/api-keys.adoc

```diff
@@ -88,13 +90,18 @@ link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas
 [method]#POST# [uri]#/api/api-key/#
 --
 
+==== Request Body
+
+[.api]
+[field]#sourceKeyId# [type]#[UUID]# [required]#Required#::
+The optional Id of an existing API Key from which a copy will be made.
```

Comment:
I'm not sure which is correct. The line above says it's required, but this says optional. If they are making a copy of a key, I would say it is required.

```suggestion
The Id of an existing API Key from which a copy will be made.
```